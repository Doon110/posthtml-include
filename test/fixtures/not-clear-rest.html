<section>
  <h1>Contact</h1>
  <include src="./test/fixtures/includes/4.html"
    locals='{"foo":"Bar"}'></include>
</section>
