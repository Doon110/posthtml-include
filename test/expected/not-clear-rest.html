<section>
  <h1>Contact</h1>
  <div>
  <h2>Bar</h2>
  <p>
    <a href="callto:+123456">+123456</a>
    <br>
    <a href="mailto:email@company.com">email@company.com</a>
  </p>
</div>

</section>
